### ISSUE TITLE
Skip virtualenv folder during execution

### ISSUE THREAD

**Issue by anapaulagomes** [jllorencetti/pytest-deadfixtures#2]
I'd like to suggest skipping the virtualenvs folders during execution or from folders that are not the project folder. Currently, I got other fixtures that are on my virtualenv. For instance:

```
Fixture name: celery_session_app, location: /Users/xxx/.virtualenvs/an-awesome-project/lib/python3.6/site-packages/celery/contrib/pytest.py:56
    Session Fixture: Return app for session fixtures.
Fixture name: celery_session_worker, location: /Users/xxx/.virtualenvs/an-awesome-project/lib/python3.6/site-packages/celery/contrib/pytest.py:77
    Session Fixture: Start worker that lives throughout test suite.
Fixture name: celery_enable_logging, location: /Users/xxx/.virtualenvs/an-awesome-project/lib/python3.6/site-packages/celery/contrib/pytest.py:94
    You can override this fixture to enable logging.
```

Just to make it clear: I'm not using those fixtures on my codebase.

**Comment by jllorencetti** [comment]
Makes sense, I'll try to take a look at it soon

**Comment by jairhenrique** [comment]
#6

**Comment by jllorencetti** [comment]
Thanks @jairhenrique!
I made a quick participation in a comment thread there.

**Comment by jllorencetti** [comment]
fixed in #6
